static/src/javascripts/projects/commercial/modules/cmp/ccpa-cmp.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+import { isInUsa } from 'common/modules/commercial/geo-utils';
+import config from 'lib/config';
+
+let ccpaApplicable;
+
+export const isCcpaApplicable = (): boolean => {
+    if (typeof ccpaApplicable === 'undefined') {
+        ccpaApplicable = isInUsa() && config.get('switches.ccpaCmpUi', true);
```

Comment:
i think `config.get('switches.ccpaCmpUi', true)` means it will return true if the switch is undefined, which seems to be the opposite of the switch logic?

Reply:
I took this to mean that if the switch isn't defined then we return `true` which is the safe option, right? We then show the CCPA CMP UI to US residents by default

Reply:
just checked with dotcom – i think the safe state should probably be `off`, because i think we want to enable it for everyone by choice, rather than disable it. 

i think you're right that the way you've written it, the same logic applies to both, but the logic should be inverted?

so the absence of the switch maintains the status quo, with only a present, enabled switch putting CCPA out to everyone

we probably want to invert _that_ when we remove the AB test after it's gone live, but maybe not yet?

Reply:
Okay, so this will mean
```suggestion
        ccpaApplicable = isInUsa() && config.get('switches.ccpaCmpUi', false);
```
As well as setting the feature switch safe state to `off` meaning that we're matching the logic here in both locations. And then after the AB test is finished, we invert that logic?